ddl/ddl_api.go

```diff
@@ -253,14 +253,24 @@ func buildColumnAndConstraint(ctx sessionctx.Context, offset int,
 
 // checkColumnCantHaveDefaultValue checks the column can have value as default or not.
 // Now, TEXT/BLOB/JSON can't have not null value as default.
```

Comment:
Update comment... in strict mode, TEXT/BLOB/JSON can't have not null value as default.

Reply:
IMHO, this function signature is very strange:
why `checkColumnCantHaveDefaultValue` return a boolean value `hasDefaultValue`

Reply:
Do agree with tiancaiamao.